src/configs.rs

```diff
@@ -47,3 +63,44 @@ pub const FIXED_SEED: [u8; 32] = [
     89, 15, 245, 34, 234, 120, 17, 218, 167, 20, 216, 9, 59, 62, 123, 217, 29, 137, 138, 115, 62,
     152, 136, 135, 48, 127, 151, 205, 40, 7, 51, 131,
 ];
+
+/// This is a config set handler for the False Positive Rate and Tightening Ratio configs.
+pub fn on_string_config_set(
+    config_ctx: &ConfigurationContext,
+    name: &str,
+    val: &'static ValkeyGILGuard<ValkeyString>,
+) -> Result<(), ValkeyError> {
+    let v = val.get(config_ctx);
+    let value_str = v.to_string_lossy();
+
+    let value = match value_str.parse::<f64>() {
+        Ok(v) => v,
+        Err(_) => {
+            return Err(ValkeyError::Str("Invalid floating-point value"));
+        }
+    };
+
+    match name {
+        "bloom-fp-rate" => {
+            if !(BLOOM_FP_RATE_MIN..=BLOOM_FP_RATE_MAX).contains(&value) {
+                return Err(ValkeyError::Str(utils::ERROR_RATE_RANGE));
+            }
+            let mut fp_rate = BLOOM_FP_RATE_F64
+                .lock()
+                .expect("We expect the fp_rate static to exist.");
+            *fp_rate = value;
+            Ok(())
+        }
+        "bloom-tightening-ratio" => {
+            if !(TIGHTENING_RATIO_MIN..=TIGHTENING_RATIO_MAX).contains(&value) {
```

Comment:
This will allow invalid values for tightening ratio (e.g 0.0 and 1.0) which should be rejected instead.

Can you change this to something like this?
```suggestion
            if !(TIGHTENING_RATIO_MIN > value  && value  << TIGHTENING_RATIO_MAX) {
```

Reply:
I will be removing = to for the range i.e. use
```
if !(BLOOM_TIGHTENING_RATIO_MIN..BLOOM_TIGHTENING_RATIO_MAX).contains(&value) {
```

When I use this
```
 if !(BLOOM_TIGHTENING_RATIO_MIN > value && value < BLOOM_TIGHTENING_RATIO_MAX) {
```
it throws the following error
```
error: right-hand side of `&&` operator has no effect
   --> src/configs.rs:101:17
    |
101 |             if !(BLOOM_TIGHTENING_RATIO_MIN > value && value < BLOOM_TIGHTENING_RATIO_MAX) {
    |                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |
note: `if `BLOOM_TIGHTENING_RATIO_MIN > value` evaluates to true, value < BLOOM_TIGHTENING_RATIO_MAX` will always evaluate to true as well
   --> src/configs.rs:101:53
    |
101 |             if !(BLOOM_TIGHTENING_RATIO_MIN > value && value < BLOOM_TIGHTENING_RATIO_MAX) {
```

Reply:
This check is incorrect as it allows 0.0 which is an invalid fp rate. 


Correct check:
```
if !(value > BLOOM_TIGHTENING_RATIO_MIN  && value < BLOOM_TIGHTENING_RATIO_MAX) {
 ```